Integration/call_integration.pro

```diff
@@ -71,6 +74,7 @@ PRO call_integration, sel_chrom, sel_name, PLOT=plot, FIX_XYRANGE=fix_xyrange, M
   IF FINITE(chk_noise) NE 1 THEN chk_noise = 0. ; set zero if not calculated yet
 
   verbose = 0
+  upsample =  1
```

Comment:
should this be removed?

Reply:
Yes, variable is not needed. This was a line remaining form tests.